cmd/kubeadm/app/apis/kubeadm/validation/validation_test.go

```diff
@@ -487,3 +489,65 @@ func TestValidateIgnoreChecksErrors(t *testing.T) {
 		}
 	}
 }
+
+func TestValidateKubeletConfiguration(t *testing.T) {
```

Comment:
note: the test case is mainly stolen from the original test case of kubelet validation.

Reply:
that's ok I guess :)